mapentity/views/generic.py

```diff
@@ -356,6 +357,34 @@ def form_invalid(self, form):
         return super().form_invalid(form)
 
 
+class MapEntityDuplicate(ModelViewMixin, SingleObjectMixin, View):
+    http_method_names = ['post', ]
+
+    @classmethod
+    def get_entity_kind(cls):
+        return mapentity_models.ENTITY_DUPLICATE
+
+    @view_permission_required()
+    def dispatch(self, *args, **kwargs):
+        return super().dispatch(*args, **kwargs)
```

Comment:
if you use view_permission_required on ost method, it's not needed to override dispatch